### PR TITLE
functionality to show/hide reviewers

### PR DESCRIPTION
**Pull request by chris-lawton**

### CODE REVIEW

File: opentech/apply/funds/templates/funds/applicationsubmission_admin_detail.html [hunk 1]
```diff
@@ -17,6 +17,7 @@
         <h5>Reviews &amp; assignees</h5>
 
         {% include 'funds/includes/review_table.html' %}
+        <a class="link link--bold link--underlined link--toggle-reviewers js-toggle-reviewers" href="#">all assigned advisors</a>
```
Comment (zerolab) [inline opentech/apply/funds/templates/funds/applicationsubmission_admin_detail.html]: Should only show if there are advisors assigned

**Comment by hippogriffic** [comment]
lgtm

File: opentech/apply/funds/templates/funds/includes/review_table.html [hunk 2]
```diff
@@ -15,8 +15,13 @@
...
 
-        {% include 'funds/includes/review_table_row.html' with reviews=reviewer_reviews %}
-        {# Add logic for showing missing reviews #}
-        {% include 'funds/includes/review_table_row.html' with reviews=object.reviewers_not_reviewed missing=True %}
+        {% if reviewer_reviews or object.reviewers_not_reviewed  %}
+            {% include 'funds/includes/review_table_row.html' with reviews=reviewer_reviews %}
+            {# Add logic for showing missing reviews #}
+            {% include 'funds/includes/review_table_row.html' with reviews=object.reviewers_not_reviewed missing=True class="hidden" %}
+            <tr><td colspan="4"><a class="link link--bold link--underlined js-toggle-reviewers" href="#">all assigned advisors</a></td></tr>
+            {% else %}
```
Comment (todd-dembrey) [inline opentech/apply/funds/templates/funds/includes/review_table.html]: Dedent the else (please)

File: opentech/apply/funds/templates/funds/includes/review_table.html [hunk 3]
```diff
@@ -15,8 +15,13 @@
     {% if object.stage.has_external_review %}
         <tr class="tr tr--subchild"><td colspan="4"><hr></td></tr>
 
-        {% include 'funds/includes/review_table_row.html' with reviews=reviewer_reviews %}
-        {# Add logic for showing missing reviews #}
-        {% include 'funds/includes/review_table_row.html' with reviews=object.reviewers_not_reviewed missing=True %}
+        {% if reviewer_reviews or object.reviewers_not_reviewed  %}
+            {% include 'funds/includes/review_table_row.html' with reviews=reviewer_reviews %}
+            {# Add logic for showing missing reviews #}
```
Comment (todd-dembrey) [inline opentech/apply/funds/templates/funds/includes/review_table.html]: remove comment

File: opentech/apply/funds/templates/funds/includes/review_table.html [hunk 4]
```diff
@@ -15,8 +15,12 @@
     {% if object.stage.has_external_review %}
         <tr class="tr tr--subchild"><td colspan="4"><hr></td></tr>
 
-        {% include 'funds/includes/review_table_row.html' with reviews=reviewer_reviews %}
-        {# Add logic for showing missing reviews #}
-        {% include 'funds/includes/review_table_row.html' with reviews=object.reviewers_not_reviewed missing=True %}
+        {% if reviewer_reviews or object.reviewers_not_reviewed  %}
+            {% include 'funds/includes/review_table_row.html' with reviews=reviewer_reviews %}
+            {% include 'funds/includes/review_table_row.html' with reviews=object.reviewers_not_reviewed missing=True class="hidden" %}
```
Comment (zerolab) [inline opentech/apply/funds/templates/funds/includes/review_table.html]: shouldn't this and the toggle be wrapped in an if?
i.e. if there no more people who did not review, then there is no need for the toggle

Comment (zerolab) [reply opentech/apply/funds/templates/funds/includes/review_table.html]: other than this, looks good to me

**Pull request merged**